Lexic.txt

```diff
@@ -0,0 +1,40 @@
+Lexic.txt
+
+1. Alphabet:
+
+a. Lowercase and uppercase letters (a-z) of English alphabet;
+b. Decimal digits (0-9);
+c. Underline character '_';
+
+2. Lexic:
+
+I. Special symbols
+
+a. operators + - * / < > <= >= == != % // && ||
+b. separators () [] {} : ;
+c. reserved words:
+   Int Char String Array const if else for while trace input var
+
+
+II. Identifiers
+
+a. sequence of letters and digits, first character must be a letter
+
+identifier := letter {extra_character}
+extra_character := "a"|...|"z" | "A"|...|"Z" | "0"|...|"9"
+
+III. Constants
+
+a. Int
+  sign := "+"|"-".
+  non_zero_digit := "1"|..."9"
+  digit:="0"|non_zero_digit
+  integer:="0"|[sign] non_zero_digit {digit} 
+b. Char
+  character:='letter'|'digit'
+c. String
+  constchar:="string"
+  string:={char}
+  char:=letter|digit
+  letter:=a|...|A|z|...|Z

```

Comment:
I think it should have been "a|...|z|A|...|Z"

Reply:
Good observation